xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1083,9 +1085,7 @@ void CRenderManager::PrepareNextRender()
   double frameOnScreen = m_dvdClock.GetClock();
   double frametime = 1.0 / g_graphicsContext.GetFPS() * DVD_TIME_BASE;
 
-  // correct display latency
-  // internal buffers of driver, assume that driver lets us go one frame in advance
-  double totalLatency = DVD_SEC_TO_TIME(m_displayLatency) - DVD_MSEC_TO_TIME(m_videoDelay) + 2* frametime;
```

Comment:
not sure what kind of a relict that is: https://github.com/xbmc/xbmc/search?utf8=%E2%9C%93&q=m_fRefreshRate&type=
don't use it